android/src/main/java/com/dieam/reactnativepushnotification/dst/TimezoneChangeReceiver.java

```diff
@@ -0,0 +1,16 @@
+package com.dieam.reactnativepushnotification.dst;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+public class TimezoneChangeReceiver extends BroadcastReceiver {
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        final String action = intent.getAction();
+        if (action.equals(Intent.ACTION_TIMEZONE_CHANGED) || action.equals(Intent.ACTION_TIME_CHANGED)) {
```

Comment:
Are we not considering DATE_CHANGED?

Reply:
DATE_CHANGED call back is not firing, only ACTION_TIME_CHANGED is firing. we only need a callback when a user updates the date. ACTION_TIME_CHANGED will fire if a user changes the system time/date.